sdk/nodejs/stackReference.ts

```diff
@@ -110,6 +118,23 @@ export class StackReference extends CustomResource {
         return promiseResult(out.promise());
     }
 
+    async isSecretOutput(name: Output<string>): Promise<boolean> {
```

Comment:
i think TS is public by default.  wheras i think you want this to be an internal helper right?  if you didn't intend to make this available for others (in which case it should hav a doc comment), it might be best to move this to a non-exported helper function below.  then it can't ever accidentally be used by anyone else.

Reply:
`isSecretOutput` was also confusing as a name.  Because it seems like it's asking if name.isSecret would be true.  which is not the case.  perhaps isSecretOutputName.

Reply:
Pulled it out and renamed to `isSecretOutputName`